release-notes.md

```diff
@@ -70,6 +70,12 @@ The following new conventions are applied to spring boot apps v2.6 and later:
 - Add Kubernetes liveness and readiness probes by using spring boot health endpoints.
 - Change management port from 8080 to 8081 to increase security of the management port.
 
+#### Supply Chain Security Tools – Scan
```

Comment:
This should say "Sign" instead of "Scan"

Reply:
Created https://github.com/pivotal/docs-tap/pull/995 to address